### PR TITLE
write document api show list posts

### PR DESCRIPTION
**Pull request by ghost**
## Description
write document api show list posts
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Comment by at-dungle** [comment]
LGTM

File: docs/api-references/post.md [hunk 1]
```diff
@@ -0,0 +1,761 @@
+## POST - API
+
+### `GET` Post
+```
+/api/products{product}/posts
```
Comment (at-tuanphan) [inline docs/api-references/post.md]: /api/products/{product_id}/posts

File: docs/api-references/post.md [hunk 2]
```diff
@@ -0,0 +1,761 @@
...
+| Key | Value |
+|---|---|
+|Accept|application\json
+
+#### Query Param
+| Param | Type | Description |
+|---|---|---|
+| sort | string | Sort Product |
+| order | string | Type Sort Product |
+| type | int | Get Product By Type |
```
Comment (at-tuanphan) [inline docs/api-references/post.md]: ở đây thêm cái `product_id` nửa em, và thêm column `require`

File: docs/api-references/post.md [hunk 3]
```diff
@@ -0,0 +1,761 @@
...
+
+##### Example
+| URL | Description |
+|---|---|
+| /api/products{product}/posts?sort=updated_at&order=desc | Get Post and sort by updated_at |
```
Comment (at-tuanphan) [inline docs/api-references/post.md]: merge 2 cái thành 1 đi em

File: docs/api-references/post.md [hunk 4]
```diff
@@ -0,0 +1,761 @@
...
+{
+    "meta": {
+        "status": "Successfully",
+        "code": 200
+    },
+    "data": {
+        "current_page": 1,
+        "data": [
+            {
+                "id": 1,
```
Comment (at-tuanphan) [inline docs/api-references/post.md]: làm 1 - 2 cái example thôi em để dài quá

**Review by ghost — COMMENTED** [review]
please check my comments

File: docs/api-references/post.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+|---|---|---|---|
+| product | int | required | Product's id |
+| sort | string | optional | Sort Product |
+| order | string | optional | Type Sort Product |
+| type | int | optional | Get Product By Type |
+
+##### Example
+| URL | Description |
+|---|---|
+| /api/products{product}/posts?sort=updated_at | Get Post and sort by updated_at |
```
Comment (at-tuanphan) [inline docs/api-references/post.md]: /api/products{product}/posts?sort=updated_at&type=1
remove dòng dưới đi

**Review by ghost — COMMENTED** [review]
please check my comment

**Review by ghost — COMMENTED** [review]
LGTM

**Pull request merged**